src/components/ProgressCircular/index.js

```diff
@@ -49,4 +54,5 @@ ProgressCircular.defaultProps = {
     assistiveText: '',
     className: undefined,
     style: undefined,
+    color: '',
```

Comment:
Adding a new prop to a new component needs to be discussed carefully, customizing the color of a component should be through theme